app/src/main/res/layout/activity_hello.xml

```diff
@@ -5,20 +5,21 @@
     android:id="@+id/helloactivity_mainframe"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:visibility="visible">
+    android:visibility="visible"
+    tools:context="org.glucosio.android.activity.HelloActivity">
 
     <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
This change was proposed by Studio Lint. Should improve performance on the ScrollView.